examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -279,6 +314,9 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
         if (ws.readyState !== ws.OPEN || typeof(parameters.keywords) === "undefined" || parameters.keywords == null || parameters.keywords.length == 0)
           return;
 
+        // for the time-series histogram
+        handleByTimeRequest(parameters, parameters.geoIds);
```

Comment:
We should not call this `handleByTimeRequest` function in any conditions, since in the following switch-case blocks, when `countmap` you also send `byTimeRequest` in the batch query. Or otherwise, you may need to get rid of those `byTimeRequest` queries in the batches.